### PR TITLE
Command fsm

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

**Comment by krystian-hebel** [comment]
@miczyg1 please don't merge until `funding` is merged.

File: regs_tb.v
Comment (miczyg1) [inline regs_tb.v]: Maybe a constant/variable to define the size?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: regs_tb.v
Comment (krystian-hebel) [reply regs_tb.v]: Done.

**Comment by macpijan** [comment]
I just merged funding branch

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**
